.circleci/config.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - run:
           name: Install Python deps in a venv
           command: |
+            sudo apt-get --allow-releaseinfo-change update
```

Comment:
@arvindsrikantan Why did you give up on adding this to the circleCI Dockerfile?

Reply:
We're using the base circleci image everywhere, so I chose to add it directly